setup.py

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env python3
+# encoding=utf-8
+
+from setuptools import setup
+
+setup(setup_requires=['pbr'], pbr=True)
```

Comment:
Am curious what `pbr` does?

Reply:
It assisted with versioning. For example it will use git tags to version the sdist if they're used. It's not essential but it makes CI pipelines cleaner